gwpy/astro/range.py

```diff
@@ -180,46 +215,46 @@ def inspiral_range(psd, snr=8, mass1=1.4, mass2=1.4, fmin=None, fmax=None,
 
     Examples
     --------
-    Grab some data for LIGO-Livingston around GW150914 and generate a PSD
+    Grab some data for LIGO-Livingston around GW150914 and generate a PSD:
 
     >>> from gwpy.timeseries import TimeSeries
     >>> hoft = TimeSeries.fetch_open_data('H1', 1126259446, 1126259478)
     >>> hoff = hoft.psd(fftlength=4)
 
-    Now we can calculate the :func:`inspiral_range`:
+    Now, we can calculate the :func:`inspiral_range`:
 
     >>> from gwpy.astro import inspiral_range
     >>> r = inspiral_range(hoff, fmin=30)
     >>> print(r)
     70.4612102889 Mpc
     """
-    mass1 = units.Quantity(mass1, 'solMass').to('kg')
-    mass2 = units.Quantity(mass2, 'solMass').to('kg')
-    mtotal = mass1 + mass2
-
-    # compute ISCO
-    fisco = (constants.c ** 3 / (constants.G * 6**1.5 * pi * mtotal)).to('Hz')
-
+    fisco = _get_isco_frequency(mass1, mass2)
     # format frequency limits
-    fmax = units.Quantity(fmax or fisco, 'Hz')
+    fmin = units.Quantity(fmin or psd.df.value, 'Hz')  # avoid DC value
```

Comment:
I think we should still be using `psd.df` here in case the units aren't `Hz`.

Reply:
Turns out you were right about this, this has been fixed in the latest commit